### PR TITLE
Added check for Python files format

### PR DESCRIPTION
**Pull request by nizamutdinovb**
Issue https://github.com/reboot-dev/pyprotoc-plugin/issues/15
This PR needs to be merged after the PR https://github.com/3rdparty/dev-tools/pull/45 is merged.

### CODE REVIEW

File: .github/workflows/check-format-style.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  check-format-style:
+    uses: 3rdparty/dev-tools/.github/workflows/check-format-style.yml@main
```
Comment (benh) [inline .github/workflows/check-format-style.yml:13]: Am I missing a PR that introduces this?

Comment (nizamutdinovb) [reply .github/workflows/check-format-style.yml:13]: Currently I am working on PR to `dev-tools` which will include the appropriate changes.
After I confirm it works I will send it to review and amend description of this PR with link to it.

Comment (nizamutdinovb) [reply .github/workflows/check-format-style.yml:13]: @benh here is PR to `dev-tools` that introduces Python format check https://github.com/3rdparty/dev-tools/pull/45.
It needs to be merged before current one.

**Comment by rjhuijsman** [comment]
Obsoleted by https://github.com/reboot-dev/pyprotoc-plugin/pull/26

**Pull request closed without merging**